apps/penxle.com/src/routes/editor/Header.svelte

```diff
@@ -845,7 +845,9 @@
           window.location.href = `/editor/${post.permalink}`;
         }}
       >
-        <p class="body-16-b mb-1 truncate">{post.draftRevision.title}</p>
+        <p class={clsx('body-16-b mb-1 truncate', post.draftRevision.title || 'text-secondary')}>
```

Comment:
P2: 클래스네임으로 항상 제목이 들어가는 건 특별한 용도로 쓰이지 않은 한 피합시다.

```suggestion
        <p class={clsx('body-16-b mb-1 truncate', post.draftRevision.title.trim().length === 0 && 'text-secondary')}>
```